src/Symfony/Bridge/Twig/Extension/HttpKernelExtension.php

```diff
@@ -28,7 +28,7 @@ public function getFunctions(): array
             new TwigFunction('render', [HttpKernelRuntime::class, 'renderFragment'], ['is_safe' => ['html']]),
             new TwigFunction('render_*', [HttpKernelRuntime::class, 'renderFragmentStrategy'], ['is_safe' => ['html']]),
             new TwigFunction('fragment_uri', [HttpKernelRuntime::class, 'generateFragmentUri']),
```

Comment:
I suppose you didn't use the first class callable syntax for this helpers to avoid loading the class if it's not used?

Reply:
Wrong guess 😅, I search only for this pattern `"self::class.'::"` and "static::class.'::"

Reply:
Should I update this too?

Reply:
nope, because that would break autoload laziness

Reply:
No. Those methods are not static and thus those arrays are not valid callables. And that's on purpose. 😉

Reply:
This cannot work actually, because this is not a real callable. It's a custom reference for this DIC loader